samples/csharp/getting-started/Ranking_AutoML/Ranking/Ranking/Program.cs

```diff
@@ -0,0 +1,232 @@
+﻿using Common;
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.Data;
+using Ranking.DataStructures;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http.Headers;
+
+namespace Ranking
+{
+    class Program
+    {
+        const string AssetsPath = @"../../../Assets";
+        const string TrainDatasetUrl = "https://aka.ms/mlnet-resources/benchmarks/MSLRWeb10KTrain720kRows.tsv";
+        const string ValidationDatasetUrl = "https://aka.ms/mlnet-resources/benchmarks/MSLRWeb10KValidate240kRows.tsv";
+        const string TestDatasetUrl = "https://aka.ms/mlnet-resources/benchmarks/MSLRWeb10KTest240kRows.tsv";
+
+        readonly static string InputPath = Path.Combine(AssetsPath, "Input");
+        readonly static string OutputPath = Path.Combine(AssetsPath, "Output");
+        readonly static string TrainDatasetPath = Path.Combine(InputPath, "MSLRWeb10KTrain720kRows.tsv");
+        readonly static string ValidationDatasetPath = Path.Combine(InputPath, "MSLRWeb10KValidate240kRows.tsv");
+        readonly static string TestDatasetPath = Path.Combine(InputPath, "MSLRWeb10KTest240kRows.tsv");
+        readonly static string ModelPath = Path.Combine(OutputPath, "RankingModel.zip");
+
+        private static uint ExperimentTime = 60;
```

Comment:
How many models are created in 60 seconds? Likely better w/ longer runtime.

```suggestion
        // Runtime should allow for the sweeping to plateau, which begins near iteration 60
        private static uint ExperimentTime = 600;
```